hyperspy/tests/signal/test_tools.py

```diff
@@ -872,7 +872,6 @@ def test_spikes_removal_tool():
     sr.find()
     assert s.axes_manager.indices == (0, 1)
     sr.threshold = 0.5
-    assert s.axes_manager.indices == (0, 0)
```

Comment:
This was testing that changing the threshold sets the indices back to zero, why removing it?